src/gui/MainWindow.cpp

```diff
@@ -378,6 +378,11 @@ void MainWindow::openRecentDatabase(QAction* action)
 void MainWindow::clearLastDatabases()
 {
     config()->set("LastDatabases", QVariant());
+    bool inWelcomeWidget = (m_ui->stackedWidget->currentIndex() == 2);
```

Comment:
I've never been a fan of hardcoded index values like this. Much prefer defines to be more explicit. What do you think?